Message/Handler/ResolveCacheHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Message\Handler;
+
+use Liip\ImagineBundle\Imagine\Filter\FilterManager;
+use Liip\ImagineBundle\Message\ResolveCache;
+use Liip\ImagineBundle\Service\FilterService;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+/** @experimental */
+class ResolveCacheHandler implements MessageHandlerInterface
+{
+    /** @var FilterManager */
+    private $filterManager;
+
+    /** @var FilterService */
+    private $filterService;
+
+    public function __construct(FilterManager $filterManager, FilterService $filterService)
+    {
+        $this->filterManager = $filterManager;
+        $this->filterService = $filterService;
+    }
+
+    public function __invoke(ResolveCache $message): void
+    {
+        $filters = $message->getFilters() ?: array_keys($this->filterManager->getFilterConfiguration()->all());
+        $path = $message->getPath();
+
+        foreach ($filters as $filter) {
+            if ($message->isForce()) {
+                $this->filterService->bustCache($path, $filter);
```

Comment:
likely needs to be adjusted to the change in https://github.com/liip/LiipImagineBundle/pull/1347 once that is merged.

Reply:
Ok, I'll change it then.